packages/react-component-library/src/components/index.ts

```diff
@@ -1,21 +1,20 @@
-// Components
 import Avatar from './Avatar'
 import Badge from './Badge'
 import Breadcrumbs from './Breadcrumbs'
 import Button from './Button'
 import Checkbox from './Checkbox'
 import Link from './Link'
 import Nav from './Nav'
+import Pagination from './Pagination'
```

Comment:
Going forward, not here, it seems Typescript favours not using `export default` and allows something like:

`
Button.tsx

export const Button = () => {}

---

index.ts

export * from './Button'

`

This gets around the having to import and then export and also makes it easier to include things like type definitions in a component and have them automatically exported too.